main.go

```diff
@@ -92,6 +96,7 @@ func main() {
 	flag.StringVar(&config.FlowlogsPipelineImage, "flowlogs-pipeline-image", "quay.io/netobserv/flowlogs-pipeline:main", "The image of Flowlogs Pipeline")
 	flag.StringVar(&config.ConsolePluginImage, "console-plugin-image", "quay.io/netobserv/network-observability-console-plugin:main", "The image of the Console Plugin")
 	flag.BoolVar(&config.DownstreamDeployment, "downstream-deployment", false, "Either this deployment is a downstream deployment ot not")
+	flag.BoolVar(&enableHTTP2, "enable-http2", enableHTTP2, "If HTTP/2 should be enabled for the metrics and webhook servers.")
```

Comment:
I feel it would be clearer to rename this to `useHTTP2` with description `Use HTTP/2 instead of HTTP/1.1 for metrics and webhook servers.`

Reply:
I'll blame @msherif1234 , I copy-pasted his work https://github.com/openshift/ingress-node-firewall/pull/415/files 😄

Joke aside, this won't be necessarily *using* http/2, but *supporting* it; ie. the protocol is use is driven by the client. So "enable" doesn't seem wrong in that perspective? Or maybe "allow", "support" ?

Reply:
(the discussion can be moved to this PR: https://github.com/netobserv/network-observability-operator/pull/466/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R100)